### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by tiserett**
- [DEMO LINK](https://tiserett.github.io/layout_miami/) 

finished site

### CODE REVIEW

**Review by AndriiRut — CHANGES_REQUESTED** [review]
Good job!

File: src/styles/blocks/main/exhibitions.scss [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+    @include font-normal;
+    color: $color-dark;
+    text-align: left;
+
+    @media (min-width: 1260px) {
+      font-size: 20px;
+    }
+  }
+
+  &__button {
```
[the remark this answers is not in the document]
Comment (tiserett) [reply src/styles/blocks/main/exhibitions.scss:52]: I set size according to mockup

File: src/styles/blocks/main/contact.scss [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+  }
+
+  &__address {
+    margin-bottom: 48px;
+
+    @media (min-width: 744px) {
+      max-width: 240px;
+
+      &-1 {
+        grid-area: 1 / 1 / 2 / 2;
```
[the remark this answers is not in the document]
Comment (tiserett) [reply src/styles/blocks/main/contact.scss:28]: I haven't seen videos about miami yet, so i'll leave it like this, but in future i'll use loops

File: src/styles/blocks/header/menu.scss [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+    text-transform: uppercase;
+    text-decoration: none;
+    @include hover-underline;
+  }
+
+  &__address {
+    @include flex-column;
+    gap: 12px;
+    @include font-bold;
+    color: $color-white;
```
Comment (AndriiRut) [inline src/styles/blocks/header/menu.scss:44]:
```suggestion

    gap: 12px;
    
    @include font-bold;
    color: $color-white;
```

File: src/styles/blocks/normalize.scss [hunk 4]
```diff
@@ -0,0 +1,15 @@
+* {
+  box-sizing: border-box;
+}
+
+h1,
```
Comment (AndriiRut) [inline src/styles/blocks/normalize.scss:5]: https://www.w3schools.com/cssref/css_default_values.asp

File: src/styles/blocks/main/explore.scss [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+      grid-template-columns: repeat(3, 1fr);
+      gap: 24px;
+    }
+  }
+
+  &__card {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    margin: 0 auto;
```
Comment (AndriiRut) [inline src/styles/blocks/main/explore.scss:31]: is this margin work?

File: src/styles/blocks/normalize.scss (hunk 4 above)
Comment (AndriiRut) [reply src/styles/blocks/normalize.scss:5]: don't add padding: 0 to all tags!

**Review by AndriiRut — CHANGES_REQUESTED** [review]

File: src/styles/blocks/header/menu.scss (hunk 3 above)
Comment (tiserett) [reply src/styles/blocks/header/menu.scss:44]: fixed

File: src/styles/blocks/normalize.scss (hunk 4 above)
Comment (tiserett) [reply src/styles/blocks/normalize.scss:5]: fixed

File: src/styles/blocks/main/explore.scss (hunk 5 above)
Comment (tiserett) [reply src/styles/blocks/main/explore.scss:31]: it's actually not ;(

**Review by AndriiRut — APPROVED** [review]
🔥

**Pull request closed without merging**